iceoryx2/src/node/mod.rs

```diff
@@ -364,6 +364,28 @@ impl<Service: service::Service> NodeState<Service> {
     }
 }
 
+#[allow(missing_docs)]
+impl<Service: service::Service> NodeState<Service> {
```

Comment:
I assume this must be public since this is used in the CLI but this is actually not public API. So instead of suppressing it with `#[allow(missing_docs)]` I would add here `#[doc(hidden]` so that it is not even shown in the documentation.
Also the IDE's code completion will not list it.